### DISCUSSION TITLE
Apollo

### DISCUSSION THREAD

**Discussion by miraykaracol** [sonsuzus/BilsemPython#70] [Genel]
Takima katilmak icin grubunuzu, gununuzu ve eklemek istediginiz bir sey varsa onu yazabilirsiniz.

Öyg1-Miray Karacol carsamba grubu

**Comment by D3r3n** [comment]
Selam ben deren öyg1 çarşamba gününden katılabilir miyim

**Reply by miraykaracol** [reply]
tabii

**Comment by sonsuzus** [comment]
Nazlı Yağmur Yıldırım da katılmak istiyor gruba. Github hesabı sorunlu olduğu için ben yazıyorum

**Comment by sonsuzus** [comment]
Ceylin Demirtaş'ı da bu takıma alalım. Github hesabı sorunluymuş. 05468607032
